src/NorthwindWebsite/Presentation/Views/Shared/Components/Breadcrumbs/Breadcrumbs.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace NorthwindWebsite.Presentation.Views.Shared.Components.Breadcrumps;
+
+public class Breadcrumbs : ViewComponent
+{
+    private const string NavigationStartPoint = "Home";
+    private const string HandleActionName = "Handle";
+    private const string ImageUploadActionName = "ImageUpload";
+
+    public IViewComponentResult Invoke()
+    {
+        var path = HttpContext.Request.Path.ToString();
+
+        var processedPathAsList = ProcessSpecialCases(path);
+
+        var breadcrumbs = NavigationStartPoint + string.Join(" > ", processedPathAsList);
+
+        return View("_Breadcrumbs", breadcrumbs);
+    }
+
+    private static List<string> ProcessSpecialCases(string path)
+    {
+        var pathAsArray = path.Split('/');
+
+        var pathAsList = new List<string>();
```

Comment:
Well, I woudn't say that name is quite good. Let's rename this and previous property like `originalPathSegments` and `breadcrumbPathSegments`